src/merkle/path_set.rs

```diff
@@ -345,45 +350,45 @@ mod tests {
         let g = int_to_node(7);
         let h = int_to_node(8);
 
-        let i = Rpo256::merge(&[a.into(), b.into()]);
-        let j = Rpo256::merge(&[c.into(), d.into()]);
-        let k = Rpo256::merge(&[e.into(), f.into()]);
-        let l = Rpo256::merge(&[g.into(), h.into()]);
+        let i = Rpo256::merge(&[a, b]);
+        let j = Rpo256::merge(&[c, d]);
+        let k = Rpo256::merge(&[e, f]);
+        let l = Rpo256::merge(&[g, h]);
 
-        let m = Rpo256::merge(&[i.into(), j.into()]);
-        let n = Rpo256::merge(&[k.into(), l.into()]);
+        let m = Rpo256::merge(&[i, j]);
+        let n = Rpo256::merge(&[k, l]);
 
         let root = Rpo256::merge(&[m.into(), n.into()]);
 
         let mut set = MerklePathSet::new(3);
 
         let value = b;
         let index = 1;
-        let path = MerklePath::new([a.into(), j.into(), n.into()].to_vec());
+        let path = MerklePath::new([a, j, n].to_vec());
         set.add_path(index, value, path.clone()).unwrap();
-        assert_eq!(value, set.get_leaf(index).unwrap());
-        assert_eq!(Word::from(root), set.root());
+        assert_eq!(*value, set.get_leaf(index).unwrap());
+        assert_eq!(RpoDigest::from(root), set.root());
```

Comment:
Why do we need `RpoDigest::from` here (and in other similar places)? Isn't root already an `RpoDigest`?